### PR TITLE
Add more specific err message for invalid struct

### PR DESCRIPTION
**Pull request by cryptopapi997**
Follow up from https://github.com/Ackee-Blockchain/trident/issues/160 , where the issue was camel -> kebab -> camel case conversion isn't always constant. This was hard to determine since trident didn't say which struct was the problem. Saying which one it was (and seeing that it sees it incorrectly capitalized) would've made this easy to debug, so this PR changes this. 

Maybe trident not supporting vars with multiple sequential capital letters could be specified in the docs somewhere too, but wasn't sure where the best spot for this is, so up to you guys.

### CODE REVIEW

**Comment by lukacan** [comment]
Hello @cryptopapi997 , thanks for the PR , we will have a look and get back to you later :)

File: crates/client/src/idl.rs [hunk 1]
```diff
@@ -545,7 +547,9 @@ pub fn parse_to_idl_program(name: String, code: &str) -> Result<IdlProgram, Erro
                 };
                 Some(fields.into_iter())
             })
-            .ok_or(Error::MissingOrInvalidProgramItems("instruction struct"))?;
+            .ok_or(Error::MissingOrInvalidProgramItems(
+                "instruction struct".to_string(),
```
Comment (lukacan) [inline crates/client/src/idl.rs:551]: What about also including the "instruction_struct_name" here?

**Review by lukacan — CHANGES_REQUESTED** [review]

**Comment by Ikrk** [comment]
Hi @cryptopapi997, thank you, it looks good to me!
Can you please rebase your branch on develop and change this PR to merge to develop branch?  We are currently pushing new features into develop branch and merge to master only on release. 

We might however reconsider this branching strategy in the (short) future in order to make contributions easier...

**Review by Ikrk — APPROVED** [review]

**Comment by Ikrk** [comment]
Thanks, merged!

**Pull request merged**